### PR TITLE
Project-weather-app by Anna, Fanny, Fiona

### PR DESCRIPTION
**Pull request by fannystenberg**

### CODE REVIEW

File: Code/script.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+ // All the DOM selectors stored as variables
+ const header = document.querySelector(".today-summary")
+ const aboutWeather = document.querySelector(".about-weather")
+ const weatherForecast = document.querySelector(".weather-forecast")
+ const firstCityWeather = document.querySelector(".first-city")
+ const secondCityWeather = document.querySelector(".second-city")
+
```
Comment (beemailley) [inline Code/script.js:7]: Did you find it easier to avoid IDs and just work with classes/querySelector?

File: Code/script.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+ // Global scope
+ const api = "https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=8d66acab5dd718723a370e1b64f22f8c"
```
Comment (beemailley) [inline Code/script.js:9]: I like that you've declared an API at the top.  It makes the functions further down the page look cleaner and easier to read.  Is there a reason you only declared one API here?

File: Code/script.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    <li>
+    <span id="fiveday">${fivedays}</span>
+    <span id="temp">${temp}º</span>
+    </li>
+    `
+    });
+  });
+};
+forecastSthlm();
+
```
Comment (beemailley) [inline Code/script.js:55]: It looks like your forecast will always start with the 0th array from the filtered list, is that correct?  So, if someone looks at the weather before noon, they will see today's forecast at the top of the list.  Was this a design choice?  Would you consider starting the forecast from "tomorrow"?  How would that look?

**Review by beemailley — COMMENTED** [review]
Overall, your code looks really tidy and minimal! I like how you've added the additional cities at the bottom of the page.  They look good in the design and work nicely. Very nicely done!

File: Code/script.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+      document.body.style.backgroundColor = "#F7E9B9";
+      document.body.style.color = "#2A5510";
+      aboutWeather.innerHTML += `<img class="weather-img" src="./icons/sunglasses.svg" alt="sun-glasses icon">`
+      aboutWeather.innerHTML += `<h2>Get your sunnies on. ${json.city.name} is looking rather great today.</h2>`
+    }
+  })
+}
+filterWeather();
+
+//Function to show other cities
```
Comment (emiliasaberski) [inline Code/script.js:91]: Nice feature to see other cities as well and nice layout with the icons

**Review by emiliasaberski — COMMENTED** [review]
I think you've done a great job that meets the expectations of this week's project. You've followed the given design and the site is responsive. Your Javascript looks neat and well put together. Especially like the way you used comments in the HTML, I think that's great when you work together like in this project (I'm gonna start doing that!). Great job everybody! 🌟

**Pull request closed without merging**